src/app/workflow/dag/dag.component.ts

```diff
@@ -155,6 +152,9 @@ export class DagComponent implements OnInit, AfterViewChecked, OnChanges {
         }
       }
     });
+    } else {
+      console.log('DAG is not truthy');
```

Comment:
Log message is not very useful, make it more descriptive

Reply:
Was for development purposes, I'll probably remove it altogether.